src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAvailabilityEventArgs.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace System.Net.NetworkInformation
+{
+    public class NetworkAvailabilityEventArgs : EventArgs
+    {
+        private bool _isAvailable;
```

Comment:
The problem here was we concurrently changed the same file. I split these event args out into a different file (because they were in a windows-specific file before), while you refactored it within its old location. I'll match it up with your refactoring.